src/randoop/EmptyExceptionCheck.java

```diff
@@ -0,0 +1,57 @@
+package randoop;
+
+import randoop.sequence.Execution;
+
+/**
+ * An {@code ExceptionCheck} that doesn't enforce the expectation of an
+ * exception at a particular statement. Allows for execution at the statement 
+ * to be either normal or throw an exception.
+ */
+public class EmptyExceptionCheck extends ExceptionCheck {
+  
+  private static final long serialVersionUID = 8159839339961663076L;
+  
+  /**
+   * Creates an exception check at the given statement.
+   * 
+   * @param exception  the exception thrown by statement
+   * @param statementIndex  the position of statement in sequence
+   */
+  public EmptyExceptionCheck(Throwable exception, int statementIndex) {
+    super(exception, statementIndex);
+  }
+
+  /**
+   * {@inheritDoc}
+   * Appends comment indicating that catch is being ignored.
+   */
+  @Override
+  protected void appendCatchBehavior(StringBuilder b, String exceptionClassName) {
+    b.append("  // ignoring fact that caught expected exception " + exceptionClassName);
```

Comment:
I find this a but too telegraphic; I suggest you use a few more words to make it clearer.